.github/workflows/integration_tests.yml

```diff
@@ -1118,20 +1125,28 @@ jobs:
           - SERVICE_NAME: helm-service
     env:
       SERVICE_NAME: ${{ matrix.SERVICE_NAME }}
-      META_KEPTN_KEPTN_PROJECT: keptn
+      KEPTN_INSTALLER_PATH: ~/downloads/keptn-installer.sh
     steps:
+      - name: Create downloads folder
+        run: mkdir ~/downloads
+
       - name: Cache Keptn CLI
         id: keptn-cli-cache
         uses: actions/cache@v2
         with:
-          path: /usr/local/bin/keptn
+          path: ~/downloads
           key: ${{ env.META_KEPTN_VERSION }}
 
-      - name: Install Keptn CLI for META_KEPTN
-        id: install_keptn_cli
+      - name: Download Keptn CLI for META_KEPTN
+        id: download_keptn_cli
         if: steps.keptn-cli-cache.outputs.cache-hit != 'true'
         run: |
-          curl -sL https://get.keptn.sh | KEPTN_VERSION=${{ env.META_KEPTN_VERSION }} bash
+          curl -sL https://get.keptn.sh -o ${{ env.KEPTN_INSTALLER_PATH }}
+          chmod +x ${{ env.KEPTN_INSTALLER_PATH }}
```

Comment:
This is not what you want here.

This downloads the install script, but not the Keptn CLI binary.

Please use the following URL to download the CLI binary:
```
https://github.com/keptn/keptn/releases/download/${META_KEPTN_VERSION}/keptn-${META_KEPTN_VERSION}-linux-amd64.tar.gz
```

Reply:
good catch! done ✅